src/components/Button/RadioButtonstyled.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export const Fieldset = styled.fieldset`
+	display: flex;
+	justify-content: start;
+	border: none;
+`;
+
+export const LabelRadio = styled.label`
+	display: flex;
+	padding: 20px;
+	color: white;
+`;
```

Comment:
try to put this components into separate files

Reply:
chances are, that you want to reuse your styled.label at some point, but I will leave it up to you.